charts/datarepo-api/templates/service-monitor.yaml

```diff
@@ -0,0 +1,25 @@
+# Service Monitor is a custom kubernetes resource that defines how Prometheus should scrape metrics from a service
```

Comment:
Cool!  Is this a new way to do this?

Reply:
Yeah working on filling out the description with details.

Reply:
But short answer this is the method we've always used to do this for the rest of Terra. TDR was doing it in a slightly different way. The big gain is that this will allow you to differentiate which TDR a metric is associated with when there are multiple TDRs in the same cluster ie your dev env or bees